controllers/login.js

```diff
@@ -122,7 +121,7 @@ router.get('/loginRedirect', (req, res, next) => {
 		if (isAuthenticated) {
 			return redirectAuthenticated(req, res);
 		}
-		if (FEATURE_MULTI_LOGIN_INSTANCES) {
+		if (Configuration.get('FEATURE_MULTI_LOGIN_INSTANCES')) {
```

Comment:
```suggestion
		if (Configuration.get('FEATURE_MULTI_LOGIN_INSTANCES') === true) {
```
i would explictly use the existing type here

Reply:
@tofixx This could be moved to a helper in the Configuration Class